MapboxCoreNavigation/VisualInstructionFormatter.swift

```diff
@@ -1,58 +0,0 @@
-import Foundation
-import MapboxDirections
-
-/**
- Formatter for creating visual instructions.
- */
-@objc(MBVisualInstructionFormatter)
-public class VisualInstructionFormatter: NSObject {
-    
-    let routeStepFormatter = RouteStepFormatter()
-    
-    /**
-     Creates the optimal text to be displayed given a `RouteLeg` and `RouteStep`.
-     */
-    @objc public func string(leg: RouteLeg?, step: RouteStep?) -> String? {
-        if let currentLeg = leg, let destinationName = currentLeg.destination.name, let step = step, step.maneuverType == .arrive {
-            return destinationName
-        } else if let destinations = step?.destinations {
-            return destinations.joined(separator: NSLocalizedString("DESTINATION_DELIMITER", bundle: .mapboxCoreNavigation, value: " / ", comment: "Delimiter between multiple destinations"))
-        } else if let step = step, step.isNumberedMotorway, let codes = step.codes {
-            return codes.joined(separator: NSLocalizedString("REF_DELIMITER", bundle: .mapboxCoreNavigation, value: " / ", comment: "Delimiter between route numbers in a road concurrency"))
```

Comment:
Remove `REF_DELIMITER`, `DESTINATION_DELIMITER`, and `ARRIVED_AT_DESTINATION` from the base Localizable.strings too.